tests/tox_env/test_api.py

```diff
@@ -32,3 +36,24 @@ def test_setenv_section_substitution(tox_project: ToxProjectCreator) -> None:
     project = tox_project({"tox.ini": ini})
     result = project.run()
     result.assert_success()
+
+
+@pytest.mark.parametrize(
+    ("key", "do_redact"),
+    [
+        pytest.param("ACCESS_TOKEN", True),
+        pytest.param("API_KEY", True),
+        pytest.param("DB_PASSWORD", True),
+        pytest.param("FOO", False),
```

Comment:
Missing some of the ones included into rege x, let's test all of them, and we should document all of the patterns in our docs (users shouldn't need to read code to find it out).

Reply:
I will be trying to get some from gitleak internal rules as they are clearly more mature than a list that I just come up in two days. Still, is a copy/paste as that project is not python based and we also don't want to add new dependencies (i suspect).